HTAN.model.csv

```diff
@@ -155,7 +155,7 @@ Ligation Condition,Name of ligase and condition for proximity ligation,,,,TRUE,S
 Biotin Enrichment,Whether biotin is used for enriching ligation product,"Yes, No",,,TRUE,Sequencing,,,
 DNA Input Amount,"Amount of DNA for library construction, in nanograms.",,,,TRUE,Sequencing,,,int
 Resolution,"Binning size used for generating contact matrix, in basepair.",,,,TRUE,Sequencing,,,
-Stripe Calling,"Tool used for identifying architectural stripe-forming, interaction hotspots.","MACS2, Other",,,TRUE,Sequencing,,,
+Stripe Calling,"Tool used for identifying architectural stripe-forming, interaction hotspots.","MACS2, Other",,,TRUE,Sequencing,,,list::-?\d+
 Loop Window,Binning size used for calling significant dot interactions (loops),,,,TRUE,Sequencing,,,int
 Stripe Window,"Binning size used for calling significant architectural stripes. Can be an integer or int/int, indicating bin size and sliding window size if different.","HiCCUPS, Cooltools, Other",,,TRUE,Sequencing,,,

```

Comment:
Change this to say

> Binning size used for calling significant architectural stripes. Can be an integer or comma-separated list of integers indicating bin size and sliding window size if different.

Reply:
The valid values here don't seem to be in the right attribute. Going back to the RFC to check

Reply:
Valdation rule for `Stripe Window` should be `list like :: regex search -?\d+`